docs/source/daml/intro/5_Restrictions.rst

```diff
@@ -78,16 +78,18 @@ There's also quite a lot going on inside the ``do`` block of the ``Redeem`` choi
 Time on DAML ledgers
 --------------------
 
-Each transaction on a DAML ledger has two timestamps called the *ledger effective time (LET)* and the *record time (RT)*. The ledger effective time is set by the submitter of a transaction, the record time is set by the consensus protocol.
+Each transaction on a DAML ledger has two timestamps called the *ledger effective time (LET)* and the *record time (RT)*. The ledger effective time is set by the participant, the record time is set by the consensus protocol.
```

Comment:
We have "LET" and "LT", let's pick one and change it throughout the docs (also gRPC reference, if we go with "LT").

Also:
1. I'd replace "participant" by "submitting participant" or "submitter's participant node" (there can be many).
2. I'd say "set by the ledger", as the ledger doesn't have to have a consensus protocol

Reply:
Changed "LET" to "LT".